package.json

```diff
@@ -6,15 +6,16 @@
     "ts-mocha": "^10.0.0",
     "ts-proto": "^1.143.0",
     "typedoc": "^0.24.7",
-    "typescript": "^5.0.2"
+    "typescript": "^5.0.2",
+    "c8": "^8.0.1"
   },
   "workspaces": [
     "./packages/*"
   ],
   "scripts": {
     "docs": "typedoc",
-    "test": "npm run test --workspaces",
-    "test-except-broken-links": "npm run test --workspace=@google-cloud/synthetics-sdk-api && npm run test --workspace=@google-cloud/synthetics-sdk-mocha"
+    "test": "c8 npm run test --workspaces",
+    "test-except-broken-links": "c8 npm run test --workspace=@google-cloud/synthetics-sdk-api && c8 npm run test --workspace=@google-cloud/synthetics-sdk-mocha"
```

Comment:
I was thinking about this a bit more - do you think that it may be better to name this "test-node-14" or something?

Reply:
I think test-node-14 is cleaner but doesn't explain (without looking into code) that we are not testing one of the packages.

Reply:
Ok, I guess let's see how this scales. If we have a few other packages that only run in certain settings, then we're going to have similarly named scripts.

Reply:
I don't feel strongly so I'd be happy to change it now?

edit:
To my knowledge there is no way to exclude a package, rather we specify which workspaces to include so I think rather than calling it `test-except-broken-links` it would be more accurate to say `test-mocha-and-api`. Otherwise in the future it is possible others may think that a new package is being tested when it isnt